### PR TITLE
Force AT to use debug logging

### PR DESCRIPTION
**Pull request by rain-on**
To aid with diagnostics of flakey ATs, additional logging is being produced.

### CODE REVIEW

File: acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java [hunk 1]
```diff
@@ -278,10 +278,11 @@ public void startNode(final BesuNode node) {
...
 
-    String level = System.getProperty("root.log.level");
-    if (level != null) {
-      params.add("--logging=" + level);
-    }
+    // String level = System.getProperty("root.log.level");
+    // if (level != null) {
+    //  params.add("--logging=" + level);
+    // }
+    params.add("--logging=TRACE");
```
Comment (RatanRSur) [inline acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java:285]: What's the advantage of doing it this way over, say, changing the `.circleci/config.yaml` to have `TRACE` as the `root.log.level`?

Comment (rain-on) [reply acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java:285]: Yeah, This PR isn't meant to go in - it was just to be able to trigger CI to determine underlying cause.

Comment (RatanRSur) [reply acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java:285]: Ahh, ok

**Comment by rain-on** [comment]
Investigation use only

**Pull request closed without merging**